### ISSUE TITLE
Use system wide installed wkhtmltopdf

### ISSUE THREAD

**Issue by joa77** [fpanaccia/Wkhtmltopdf.NetCore#27]
Is there any chance to use a system wide installed version of wkhtmltopdf?
Because my hoster already deploys an up to date version on theirs servers. I have multiple services which use pdf rendering with wkhtmltopdf and it would be very nice to use the version provided by the system, instead having every app it's own binary

**Comment by fpanaccia** [comment]
Hi!, i think you could, without any modification in the library, you could do something like this in the startup

services.AddWkhtmltopdf("PATH_TO_SYSTEM_WIDE_VERSION");

**Comment by joa77** [comment]
Leider wird vom `WkhtmlDriver` vorgegeben, dass der Pfad zu `wkhtmltopdf` eine Bezeichnung des Betriebssystems enthält.
```
            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
            {
                rotativaLocation = Path.Combine(wkhtmlPath, "Windows", "wkhtmltopdf.exe");
            }
            else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
            {
                rotativaLocation = Path.Combine(wkhtmlPath, "Mac", "wkhtmltopdf");
            }
            else
            {
                rotativaLocation = Path.Combine(wkhtmlPath, "Linux", "wkhtmltopdf");
            }
```

**Comment by fpanaccia** [comment]
Yeah, i forgot about those lines, the online way i think it could work is with a few modifications

**Comment by fpanaccia** [comment]
i think im gonna add a parameter in the setup like "usesystemwide" or something like that....and invoke wkhtmltopdf, the only catch is that the path to wkhtmltopdf will have to be added to the env variable PATH

**Comment by gurustron** [comment]
@fpanaccia I'm planning to implement such feature in my fork. We can collaborate on that. I think user should be able to provide full path to executable, also I suggest removing the static path from settings.

**Comment by fpanaccia** [comment]
Hi @gurustron! yes, i had planned to do that in the weekend, i have little time available as this moment, if you have the time, please, be my guest
